frontend/src/components/Pages/FrontPage/MissionOverview/CreateMissionButton.tsx

```diff
@@ -6,8 +6,8 @@ import { useInstallationContext } from 'components/Contexts/InstallationContext'
 
 export const CreateMissionButton = (): JSX.Element => {
     const { TranslateText } = useLanguageContext()
-    const [isPopoverOpen, setIsPopoverOpen] = useState<boolean>(false)
     const { installationCode } = useInstallationContext()
+    const [isPopoverOpen, setIsPopoverOpen] = useState<boolean>(false)
```

Comment:
Are these reorderings done to group the useStates away from the useContexts? Just curious

Reply:
Yes, looks a lot better and I find it easier to read